DataFormats/SiStripCluster/src/SiStripApproximateCluster.cc

```diff
@@ -25,6 +30,39 @@ SiStripApproximateCluster::SiStripApproximateCluster(const SiStripCluster& clust
     maxSat = std::max<int>(maxSat, thisSat);
   }
   if (maxSat >= maxNSat) {
+    filter_ = true;
     isSaturated_ = true;
   }
+
+  // mimicing trimming filter used in StripSubClusterShapeTrajectoryFilter...
+  double trimMaxADC_ = 30.;
+  double trimMaxFracTotal_ = .15;
+  double trimMaxFracNeigh_ = .25;
+  double maxTrimmedSizeDiffNeg_ = .7;
+  double maxTrimmedSizeDiffPos_ = 1.;
```

Comment:
should all of these be `constexpr` and private data members of the class?

Reply:
This is fixed in a new commit. Please have a look